README.md

```diff
@@ -281,6 +282,13 @@ Navigate to `/assets/typedoc/index.html`
 
 [View instructions to use the Christmas trees sidebar template.](/wiki/development/christmas-trees-sidebar-template.md)
 
+#### Basic Auth
+The `staging-deploy` and the `prod-deploy` currently have a basic auth file generated from ENV vars in circle.
+`BASIC_AUTH_USER` and `BASIC_AUTH_PASS` which is an htpassword are copied into a frontend/src/Staticfile.auth which is used by the cloud.gov static buildpack.
```

Comment:
This could be splitting hairs since this will hopefully be removed soon anyways, but is it worth mentioning that this should be the encrypted password in the ENV var, not just a plain text password?

Reply:
yeah, i can add that its the password hash yeah.